### ISSUE TITLE
Question: compiling for Windows on Linux/RPi

### ISSUE THREAD

**Issue by miqumi** [sm64pc/sm64pc#546]
Hewwo! Just here to ask if it is possible to compile for Windows on Linux distros or RPi...

**Comment by coreybruce** [comment]
Yep just follow the same instructions on your pi and it should compile, I would properly recommend a pi 4 or pi 5, I also have compiled binaries for x64, i686 and Arm64 on my repo https://gitlab.com/supermario64pkg/binaries/951389c

**Comment by miqumi** [comment]
> follow the same instructions  

which instructions? As the linux page says this:
![image](https://github.com/sm64pc/sm64ex/assets/80510430/9b859d41-165d-4ee2-96e5-85daac926f66)
